.github/pull_request_template.md

```diff
@@ -4,7 +4,7 @@
 
 <!-- Explain what your PR does -->
 
-CHANGELOG: <!-- Insert a short, end-user understandable sentence in past tense right here, e.g.: Client: Fixed crash when clicking the connect button too fast -->
+CHANGELOG: <!-- Insert a short, end-user understandable sentence in past tense right here, e.g.: Client: Fixed crash when clicking the connect button too fast or SKIP if the change should not be documented in the ChangeLog -->
```

Comment:
This is a nice reminder....... but... we also need to make it clear when it's allowed to be used.

Ideally it wouldn't be needed and the tooling could work it out.

Reply:
That's part of https://github.com/jamulussoftware/jamulus/issues/2788 I suppose. Resolving therefore.